proto/pingpong.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package pingpong;
+
+message PingRequest {
+  bool first = 1;
+  string destination = 2;
+  string origin = 3;
+  string message = 4;
+}
+
+message PongResponse {
+  bool first = 1;
+  string destination = 2;
+  string origin = 3;
+  string message = 4;
+}
```

Comment:
I'm concerned we have a 100% similar message here, maybe there is some possible refactoring between the two, no?